lib/happo/public/happo-styles.css

```diff
@@ -1,4 +1,11 @@
 body {
   background-color: #f0f0f0;
   font-family: helvetica, arial;
+  background: repeating-linear-gradient(
+    -45deg,
+    #f0f0f0,
+    #f0f0f0 10px,
+    #f4f4f4 10px,
+    #f4f4f4 20px
+  );
```

Comment:
Also, I think it might be nice to scope this pattern to a container that contains just the image